README.md

```diff
@@ -85,8 +85,8 @@ Only tested code will be accepted. Please follow fix the style guide.
 
```

Comment:
Can you please run those for some code standard fixes?

./vendor/bin/broc-code fix src
./vendor/bin/broc-code fix tests